### ISSUE TITLE
When executing from a python file for the first time with `shift+enter`, occasionally the audio cue does not play

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#233922]
I saw @jooyoungseo reproduce this - but it sometimes doesn't occur.

Select some text in a python file, use `shift+enter`. The audio cue does not play to indicate the cell ran in the interactive window.

**Comment by amunger** [comment]
I can't seem to get this one to repro, do you remember which auto-focus setting he was using? Were they typically cells that ran instantly, or more long running stuff?

**Comment by meganrogge** [comment]
It was a cell that runs immediately like x=12.

**Comment by jooyoungseo** [comment]
@meganrogge I cannot reproduce this anymore. We can close and reopen later.
